change/@ni-nimble-components-37c84290-c5e8-4c31-a22e-e69547cc96c0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Sure, I'll show these changes to Brandon at the team meeting so the rest of the team can give input as well. The inner border you mentioned is intentional, Brandon has this in his design for the buttons in the Figma Spec.

The diffs that aren't related to the buttons are showing up because I changed the borderHoverColor attribute color for the dark theme to `PowerGreen`, as when I looked through Brandon's other Figma designs (Inputs, Checkboxes, etc.) this color seems to be the new default border color for the dark theme. I'll also double check with him about this in the team meeting.

Regarding the borders changing- this was also intentional. It had to do with issues getting the border-width css attribute to work and ultimately deciding to use only box-shadow for button borders. I'll go more into depth on this in the team meeting as well.